scripts/build.sh

```diff
@@ -0,0 +1,66 @@
+mkdir temp && cd temp
+
+npm init -y
+
+npm install --save-dev @percy/cli
```

Comment:
we'll be building of existing commit, so no install here.

Reply:
we need dist folder to convert code to cjs and use it but we don't have `bundle.js` in dom package in src folder. so I installed cli. is there another way to do so?

Reply:
```
yarn build
```

this generates the dist folder.

Reply:
thanks, fixed